assets/scripts/script.js

```diff
@@ -263,6 +189,91 @@
       });
     }
 
+    /**
+    * Initialize gallerya slider
+    */
+    function initSlider() {
+      // Adds slide navigation through variation thumbnails for variable products
```

Comment:
This is a good idea to do in general, but can you move the code back up to its original position in the file?